go.mod

```diff
@@ -5,7 +5,7 @@ go 1.24.0
 toolchain go1.24.1
 
 require (
-	github.com/ClickHouse/ch-go v0.68.0
+	github.com/ClickHouse/ch-go v0.68.1-0.20251003160956-f6cfd63ac24c
```

Comment:
FYI: This points to this [PR commit in ch-go.](https://github.com/ClickHouse/ch-go/pull/1097) Once it's merged we can point `main` (or after releasing point to specific newer version)

Reply:
Merged